### ISSUE TITLE
Change site title from orange to white

### ISSUE THREAD

**Issue by axamei** [pulibrary/pomegranate#270]
Low contrast ratio. Not accessible.

<img width="687" alt="screen shot 2017-05-26 at 3 49 11 pm" src="https://cloud.githubusercontent.com/assets/8161144/26510346/e8fe8e98-422a-11e7-8264-a68405adc4ae.png">

vs

<img width="732" alt="screen shot 2017-05-26 at 3 49 02 pm" src="https://cloud.githubusercontent.com/assets/8161144/26510347/e9009954-422a-11e7-8f72-cd30cf0ecec7.png">

**Comment by jpstroop** [comment]
👍
